### PR TITLE
Adding leftover strings to resource file for localization

### PR DESCRIPTION
**Pull request by jyvenugo**

### CODE REVIEW

File: preview/MsixCore/msixmgr/msixmgr.rc [hunk 1]
```diff
@@ -57,6 +57,34 @@ IDI_ICONBIG             ICON                    "icon1.ico"
 IDI_ICONSMALL           ICON                    "icon2.ico"
 
 
+/////////////////////////////////////////////////////////////////////////////
+//
+// Version
```
Comment (wcheng-msft) [inline preview/MsixCore/msixmgr/msixmgr.rc]: you may need to manually edit this not to change

**Comment by wcheng-msft** [comment]
>         L"Install",  // text

do these need to be localized too?

---
Refers to: preview/MsixCore/msixmgr/InstallUI.cpp:462 in c1cdcd2. [](commit_id = c1cdcd2ca64ddad33241460fc87decb928c539e5, deletion_comment = False)

**Comment by wcheng-msft** [comment]
>         L"Cancel",  // text

and this


---
Refers to: preview/MsixCore/msixmgr/InstallUI.cpp:481 in c1cdcd2. [](commit_id = c1cdcd2ca64ddad33241460fc87decb928c539e5, deletion_comment = False)

**Comment by wcheng-msft** [comment]
>         L"Installing app package",

this?

---
Refers to: preview/MsixCore/msixmgr/InstallUI.cpp:520 in c1cdcd2. [](commit_id = c1cdcd2ca64ddad33241460fc87decb928c539e5, deletion_comment = False)

**Comment by wcheng-msft** [comment]
>         L"Reason:",

this?

---
Refers to: preview/MsixCore/msixmgr/InstallUI.cpp:552 in c1cdcd2. [](commit_id = c1cdcd2ca64ddad33241460fc87decb928c539e5, deletion_comment = False)

File: preview/MsixCore/msixmgr/msixmgr.rc (hunk 1 above)
Comment (jyvenugo) [reply preview/MsixCore/msixmgr/msixmgr.rc]: Updated.

File: preview/MsixCore/msixmgr/InstallUI.cpp [hunk 2]
```diff
@@ -648,7 +648,7 @@ int UI::CreateInitWindow(HINSTANCE hInstance, int nCmdShow, const std::wstring&
 
     if (!hWnd)
     {
-        MessageBox(NULL, L"Call to CreateWindow failed!", title.c_str(), NULL);
+        MessageBox(NULL, GetStringResource(IDS_STRING_ERROR_INIT_WINDOW_FAILED).c_str(), title.c_str(), NULL);
```
Comment (wcheng-msft) [inline preview/MsixCore/msixmgr/InstallUI.cpp]: >MessageBox(NULL, GetStringResource(IDS_STRING_ERROR_INIT_WINDOW_FAILED).c_str(), title.c_str(), NULL); [](start = 8, length = 102)

actually, i think this one is not really actionable for the user.

**Pull request merged**